components/Common/Blockquote/index.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import Blockquote from './';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+type Story = StoryObj<typeof Blockquote>;
+type Meta = MetaObj<typeof Blockquote>;
+
+export const Default: Story = {
+  args: {
+    children: (
+      <>
+        <p>
```

Comment:
You don't need to pass paragraphs as children here, can you remove these <p> elements to simplify things?

MDX sometimes will make paragraph elements but not all the times